pyop2/compilation.py

```diff
@@ -369,7 +380,7 @@ class LinuxCompiler(Compiler):
     :kwarg comm: Optional communicator to compile the code on (only
     rank 0 compiles code) (defaults to COMM_WORLD)."""
     def __init__(self, cppargs=[], ldargs=[], cpp=False, comm=None):
-        opt_flags = ['-march=native', '-O3', '-ffast-math']
+        opt_flags = ['-march=native', '-O3', '-ffast-math', '-fopenmp']
```

Comment:
What do we need `-fopenmp` for at the moment?

Reply:
Nada, deleted.